WooCommerce/Classes/ViewRelated/Custom Fields/CustomFieldsListView.swift

```diff
@@ -68,6 +69,31 @@ private extension CustomFieldsListHostingController {
                 self?.saveCustomFieldButtonItem.isEnabled = hasChanges
             }
             .store(in: &subscriptions)
+
+        viewModel.$isSavingChanges
+            .sink { [weak self] isSavingChanges in
+                if isSavingChanges {
+                    self?.displayInProgressController()
+                } else {
+                    self?.dismissInProgressController()
+                }
+            }
+            .store(in: &subscriptions)
+    }
+
+    func displayInProgressController() {
+        let inProgressController = InProgressViewController(
```

Comment:
I used the `InProgressViewController` for the loading UI, we agreed on using a modal for the loading and that's what we do in Android, but if there is a better option here please let me know (like for example just using SwiftUI's `ProgressView`)